src/templates/LDCETemplate.ts

```diff
@@ -174,15 +174,20 @@ export function LDCETemplate(passport: { data: object }) {
         fields: [
           {
             name: "Please state under what grounds is the certificate sought",
-            value: "", // TODO
+            value: get("application.basisOfLawfulness"),
```

Comment:
This displays as "notDevelopment". Does Alastair need to provide a `.form` value here?

<img width="677" alt="image" src="https://user-images.githubusercontent.com/20502206/220320352-129df4c2-9b7a-477e-aa07-998cdc5a1293.png">